.devcontainer/scripts/configure-proxies.sh

```diff
@@ -96,13 +96,6 @@ if [ "${USE_PROXIES}" = "true" ]; then
         git config --global https.proxy ${HTTPS_PROXY}
     fi
 
-    echo "# Proxy settings" >> /etc/wgetrc
```

Comment:
We need to observe any regression issue with the wget due to this change. 
- But currently no action is required